fusionbackend/src/main/java/io/fusion/fusionbackend/service/draft/AssetSeriesDraftService.java

```diff
@@ -57,7 +57,10 @@ public AssetSeries fromAssetTypeTemplate(final Long targetCompanyId,
         final Company targetCompany = companyService.getCompany(targetCompanyId, false);
 
         final AssetSeries newAssetSeries = AssetSeries.builder().build();
-        newAssetSeries.copyFrom(assetTypeTemplate);
+
```

Comment:
why this change?

Reply:
copyFrom() ist Teil des Update-Mechanismus. Es sollte so und hier nicht verwendet werden. Passt das? Sonst sprechen wir nochmal darüber.